.python-version

```diff
@@ -0,0 +1 @@
+vp
```

Comment:
What's this file and contents for?

Reply:
Oh, no, that's for my local pyenv to automatically activate the env when I cd into the directory. It does not belong into the repository.